### ISSUE TITLE
real names and profile images

### ISSUE THREAD

**Issue by jancborchardt** [agical/friendsunhosted#5]
http://sharedstuff.org by @shybyte already has a profile feature where you can put in your real name and a profile picture. There should be a shared »profile« module in remoteStorage where you can put these things in. The real name and profile image should probably be public by default also.

cc @michielbdejong @nilclass regarding modules.

**Comment by brolund** [comment]
Yes, I would assume that this is a common thing in unhosted apps. I think we want a friendsunhosted profile as well, but we should definitely enable using existing data.

**Comment by jancborchardt** [comment]
Yeah, shared data is the big advantage of unhosted apps. There doesn’t need to be a »friendsunhosted profile«, it will just be the unhosted profile. Next to username and image, F#Un might use additional fields which Sharedstuff does not use yet – but the important thing is that they could.
